src/test/java/com/logigear/LoginTest.java

```diff
@@ -71,4 +74,24 @@ public void TC05() {
         Assert.assertEquals(actualMsg, Constant.MAXIMUM_LOGIN_ATTEMPTS_WARNING_MSG, "Error message fails to display!");
     }
 
+    @Test(description = "TC06 - Additional pages display once user logged in")
+    public void TC06() {
+        LoggerHelper.startTestCase("TC03 - Additional pages display once user logged in");
```

Comment:
TC06

Reply:
So sorry, that my fault. I will check it now